### PR TITLE
Fix AllDisplayLevelsSelecteded typo

### PR DESCRIPTION
**Pull request by williambohrmann3**
Deprecates `AllDisplayLevelsSelecteded` and replaces with `AllDisplayLevelsSelected`.

### CODE REVIEW

File: src/Toolkit/Toolkit.UI.Controls/FloorFilter/FloorFilter.cs [hunk 1]
```diff
@@ -673,9 +674,31 @@ public bool AllDisplayLevelsSelecteded
                 }
```
Comment (dotMorten) [inline src/Toolkit/Toolkit.UI.Controls/FloorFilter/FloorFilter.cs:674]: Duplicate implementation. The old property should forward to the new one.
```cs
public bool AllDisplayLevelsSelecteded
{
        get => AllDisplayLevelsSelected;
        set => AllDisplayLevelsSelected = value;
}
```

File: src/Toolkit/Toolkit.UI.Controls/FloorFilter/FloorFilter.cs [hunk 2]
```diff
@@ -341,7 +341,7 @@ private void HandleControllerPropertyChanges(object? sender, PropertyChangedEven
                     _autoVisibilityWrapper?.SetValue(VisibilityProperty, _controller.ShouldDisplayFloorPicker ? Visibility.Visible : Visibility.Collapsed);
                     break;
                 case nameof(_controller.AllDisplayLevelsSelected):
-                    OnPropertyChanged(nameof(AllDisplayLevelsSelecteded));
```
Comment (dotMorten) [inline src/Toolkit/Toolkit.UI.Controls/FloorFilter/FloorFilter.cs:344]: The old property is still changing too.

**Review by dotMorten — COMMENTED** [review]
Some minor comments. Also note that this binding should now work:
- https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/blob/a6ee2c2e8b217bb9eb6f62f3e3897e3b95608be8/src/Toolkit/Toolkit.WPF/UI/Controls/FloorFilter/FloorFilter.Theme.xaml#L341
- https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/blob/a6ee2c2e8b217bb9eb6f62f3e3897e3b95608be8/src/Toolkit/Toolkit.WinUI/UI/Controls/FloorFilter/FloorFilter.Theme.xaml#L556

It would be good to verify that it acts like expected

File: src/Toolkit/Toolkit.UI.Controls/FloorFilter/FloorFilter.cs (hunk 2 above)
Comment (williambohrmann3) [reply src/Toolkit/Toolkit.UI.Controls/FloorFilter/FloorFilter.cs:344]: Do you mean we want the old property to keep changing? So undo the deletion of line 344? I'm not sure I follow.

Comment (mstefarov) [reply src/Toolkit/Toolkit.UI.Controls/FloorFilter/FloorFilter.cs:344]: For the sake of backwards compatibility, for all the software that has worked around our old misspelling, we should keep firing a PCE for that old property name too &mdash; in addition to the new correctly-spelled PCE.

File: src/Toolkit/Toolkit.UI.Controls/FloorFilter/FloorFilter.cs [hunk 3]
```diff
@@ -658,7 +659,20 @@ public void SetSelectedLevelWithoutZoom(FloorLevel newLevel)
         /// <remarks>
         /// This is used for showing an entire facility in 3D.
         /// </remarks>
+        [Obsolete("This property is deprecated. Use AllDisplayLevelsSelected instead.")]
         public bool AllDisplayLevelsSelecteded
```
Comment (mstefarov) [inline src/Toolkit/Toolkit.UI.Controls/FloorFilter/FloorFilter.cs:663]: I would add `[EditorBrowsable(EditorBrowsableState.Never)]` to hide this from IntelliSense

**Review by mstefarov — APPROVED** [review]
I suggest one more attribute, but otherwise this PR looks good to me

**Comment by williambohrmann3** [comment]
@dotMorten verified bindings

**Review by dotMorten — APPROVED** [review]

**Pull request merged**